### DISCUSSION TITLE
Unsuccessful deployment

### DISCUSSION THREAD

**Discussion by jakamum** [vercel/community#6647] [Help] [answered]
### Summary

I am having trouble with updating my code without errors after the initial deployment of several months ago. I have tried many things including trying to problem solve locally(installing and uninstalling dependencies) and redeploying and starting a new project for deployment (which replicated the error), but no success. Here is the error: 
Creating an optimized production build...
Failed to compile.
[eslint] 
src/Gallery.js
Syntax error: Identifier 'Gallery' has already been declared. (20:24) (20:24)
Error: Command "npm run build" exited with 1   

What I do know is that I have no duplication of 'Gallery' in my code and the relevant files were working with the original deployment. I am at a loss as to what to do after attempting to problem-solve for days. Please note, I am new to this realm and this is my first deployment project (which, as I said, functioned well at first deployment).

### Example

https://vercel.com/jacquis-projects/aethel-music/H47c68wANAyFVYRC5wd5nRYH85T1

### Steps to Reproduce

Amend  files in GitHub repo -> commit -> error - Command "npm run build" exited with 1
Build Failed
Command "npm run build" exited with 1

Status
Error
Environment
Production
Duration
1m 20s (1d ago)
Domains
[aethel-music-git-main-jacquis-projects.vercel.app](https://aethel-music-git-main-jacquis-projects.vercel.app/)
[aethel-music-f72dyfgfy-jacquis-projects.vercel.app](https://aethel-music-f72dyfgfy-jacquis-projects.vercel.app/)
Source
[main](https://github.com/jakamum/AethelMusic/tree/main)
e3efee6
Upload schema types to aethel-music

Result:
Creating an optimized production build...
Failed to compile.
[eslint] 
src/Gallery.js
Syntax error: Identifier 'Gallery' has already been declared. (20:24) (20:24)
Error: Command "npm run build" exited with 1

**Comment by QuiiBz** [comment, marked as answer]
Based on the logs you provided, the issue seems to be an ESLint error. Do you see the same error when building your application locally?

**Reply by jakamum** [reply]
Thank you for replying. There was no reference to "Gallery' but when I tried to upload a new repo to GitHub with a view to creating a separate deployment project, there was a reference to eslint. I tried to isolate a potential duplicate of eslint. I uninstalled and reinstalled one version. It then produced an error about the uniqueness of eslint-plugin-jsx-a11y. After experimenting with trying to locate and address duplicates of configuration, I must have succeeded to a degree but the result was that, on running npx eslint. it produced thousands of warnings and errors and none of the images were accessed throughout the website. I feel like I'm stumbling around in the dark so who knows what trouble I'm causing in my attempts to address the initial problem.

**Reply by QuiiBz** [reply]
I'd advise to go back to the state of your project when you created this discussion and simply disable ESLint during the build process. That should let you deploy successfully and you can separately fix the ESLint errors.

**Reply by jakamum** [reply]
Thanks again - I really appreciate your attention. I did discover a duplicate in the GitHub Gallery code which wasn't evident in the original code - so that problem was resolved. Saying that, I have learned more about eslint, multiple versions and duplication etc that was helpful in the process. I will close this now while I go back and address the (many) previous failed commits and take things from there. This has confirmed to me the 'talking to the rubber duck theory' and also the 'it works on my machine'  experience. Believe me, I did try to solve this on my own before resorting to asking for help.

**Reply by jakamum** [reply]
Final update: I had success with a new version of the project and have successfully deployed a second project - so I guess I'm not a complete idiot after all. Thanks again for your help.
